components/x-live-blog-wrapper/src/HydratedLiveBlogWrapper.jsx

```diff
@@ -0,0 +1,17 @@
+import { h, Fragment } from '@financial-times/x-engine';
+import { LiveBlogWrapper } from "./LiveBlogWrapper";
+import { Serialiser, HydrationData } from '@financial-times/x-interaction';
+import { listenToLiveBlogEvents } from './LiveEventListener';
+
+const HydratedLiveBlogWrapper = (props) => {
+	const serialiser = new Serialiser();
+
+	return (
+		<Fragment>
+			<LiveBlogWrapper {...props} serialiser={serialiser} />
+			<HydrationData serialiser={serialiser} />
+		</Fragment>
```

Comment:
i'd originally intended that using the serialiser would be left for the consuming app, since you might want to reuse the same serialiser for multiple components to reduce the output, and you might have particular requirements for where the `HydrationData` gets output

also i don't know if we need to think coordinating this with the various Page Kit components that output server data; `x-interaction` predates Page Kit and the hydration stuff hasn't been used in production yet

happy to chat further about this! 🙂

Reply:
@apaleslimghost that makes sense. Currently, one of the consumer apps (`next-article`) is rendering this component using `x-handlebars`. I couldn't find a way to use serialiser with it. Do you think it is possible?

Would it be acceptable to offer a plain version of this component that doesn't serialise the hydration data as well as having this one as an option?

Reply:
sorry, i missed these comments. i think setting `res.locals.serialiser` will work and get passed through to the component. you might need to do `serialiser=@root.serialiser` when rendering it

Reply:
That's good to know. Thanks!